library/platform_util.c

```diff
@@ -51,9 +51,20 @@
 #include <stddef.h>
 #include <string.h>
 
-/* Max number of loops for mbedtls_platform_random_delay */
+/* Max number of loops for mbedtls_platform_random_delay. */
 #define MAX_RAND_DELAY  100
 
+/* Parameters for the linear congruential generator used as a non-cryptographic
+ * random number generator. */
+#define RAND_MULTIPLIER 1103515245
+#define RAND_INCREMENT  12345
+#define RAND_MODULUS    0x80000000
```

Comment:
Why is this 0x80000000? How were the multiplier and incrementation values picked?

Reply:
All three parameters are standard values for a rand() function used in glibc and ANSI C

Reply:
@piotr-now , lets add comment and reference to rand() function of ANSI C

Reply:
Added